scripts/lint.sh

```diff
@@ -31,4 +31,4 @@ trap moveBackBackend EXIT
 mv -f ./backend.tf{,.bak} || true
 terraform init
 # TODO: test to see if tflint is installed first.
-tflint --deep ./
+tflint ./
```

Comment:
Instead of removing this, can we create make lint and make deeplint; and call make lint in CI pipeline and make deeplint from within CD pipeline?